module1/homework/removeVowels/vowel.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+const std::string vowels{"aeiouy"};
```

Comment:
It can be a constexpr.

Reply:
Such an explanation in this case: 
Constexpr variable cannot have non-literal type 'const std::string' (aka 'const basic_string<char>') 'basic_string<char, std::char_traits<char>, std::allocator<char> >' is not literal because it is not an aggregate and has no constexpr constructors other than copy or move constructors

Reply:
You're right. The same as a std::vector.
But... C++20 will add constepxr containers like std::string and std::vector.

Cool fact:
In C++17 we can use constexpr std::string_view
(https://en.cppreference.com/w/cpp/header/string_view)